bloom/generators/debian/templates/ament_cmake/copyright.em

```diff
@@ -1,11 +1 @@
-Format: Bloom subset of https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: @(Name)
-@[if BugTracker]Upstream-Contact: @(BugTracker)@\n@[end if]@
-@[if Source]Source: @(Source)@\n@[end if]@
-@[for License, Text in Licenses]@
-
-Files: See file headers in repository for details
-Copyright: See package copyright in source code for details
-License: @(License)
- @(Text)
-@[end for]@
+@(Copyright_file_content)
```

Comment:
The capitalization of this variable name is unexpected `copyright_file_content` would be more consistent, here and in 4 other places.

Reply:
Honestly, I think for consistency, it should be capitalized. If you look at the other variables, let alone in this snippet: `Text`, `License`, ...

Reply:
Well that's unfortunate. I guess stick with what's local then.